build/mixer.sh

```diff
@@ -52,6 +54,21 @@ build_bundles() {
     # shellcheck disable=SC2086
     sudo_mixer_cmd build bundles ${mixer_opts_bundles}
     log_line
+
+    if function_exists sign_update; then
+        log_line "Injecting Swupd_Root.pem into full chroot"
+        local chroot_cert_file="/usr/share/clear/update-ca/Swupd_Root.pem"
+        local chroot_cert_dir="$(dirname "${chroot_cert_file}")"
+        local mix_ver_chroot_dir="update/image/${mix_ver}/full"
+        sudo mkdir -p "${mix_ver_chroot_dir}${chroot_cert_dir}"
+        sudo cp -f Swupd_Root.pem "${mix_ver_chroot_dir}${chroot_cert_file}"
+        sudo tee update/os-core-update-extra-files > /dev/null << EOF
```

Comment:
Code looks clearer if you append on by one instead of using here doc. And is not enough raw data to justify a heredoc.

Do like we did for latest:
" echo -n "${mix_ver}" | sudo tee update/latest > /dev/null"